config/routes.rb

```diff
@@ -215,6 +215,16 @@
     end
   end
 
+  get '/check_moving_requirements/start', to: 'check_moving_requirements#start', as: :start_check_moving_requirements
+  get '/check_moving_requirements/edit', to: 'check_moving_requirements#edit', as: :edit_check_moving_requirements
+  put '/check_moving_requirements/update', to: 'check_moving_requirements#update', as: :update_check_moving_requirements
+  get '/check_moving_requirements/result', to: 'check_moving_requirements#result', as: :result_check_moving_requirements
+
+  # This solution does not work because edit action could have an empy commodity_code in the URL
+  # resources :check_moving_requirements, only: %i[show update edit] do
+  #   get 'start', on: :collection
+  # end
+
```

Comment:
I know I'd originally suggested a singular resource but it feels like this set of screens maps better to a plural resource?

```ruby
  scope module: :green_lanes do
    resources :check_moving_goods_requirements, only: %i[index edit update destroy]
  end

Reply:
I adopted a similar solution, under the existing
`    namespace :green_lanes do`